quest/response/__init__.py

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Dont forget little strings at the top of each file to explain what they are doing.